packages/mobx-state-tree/src/types/primitives.ts

```diff
@@ -116,6 +118,44 @@ export const integer: ISimpleType<number> = new CoreType<number, number, number>
     (v) => isInteger(v)
 )
 
+/**
+ * `types.float` - Creates a type that can only contain an float value.
+ * This type is used for float values by default
```

Comment:
```suggestion
```

**nitpick:** @chakrihacker - this is not technically true, I don't think we infer `types.float` for floating point numbers. I think this comment is true of `types.number`, based on how [`getPrimitiveFactoryFromValue` behaves](https://github.com/mobxjs/mobx-state-tree/blob/85445de5f0e90d8c30545dfa008e2704d1eaebbb/packages/mobx-state-tree/src/types/primitives.ts#L224), which is what we call in [`toPropertiesObject`](https://github.com/mobxjs/mobx-state-tree/blob/85445de5f0e90d8c30545dfa008e2704d1eaebbb/packages/mobx-state-tree/src/types/complex-types/model.ts#L258) when doing type inference.

I'm going to make this edit since it's small and really more of a nitpick than anything.